niworkflows/interfaces/utils.py

```diff
@@ -77,3 +106,23 @@ def _run_interface(self, runtime):
         self._results['out_file'] = os.path.abspath("motion_params.txt")
         np.savetxt(self._results['out_file'], mpars)
         return runtime
+
+
+def _copyxform(ref_image, out_image, message=None):
+    # Read in reference and output
+    orig = nb.load(ref_image)
+    resampled = nb.load(out_image)
+
+    # Copy xform infos
+    qform, qform_code = orig.header.get_qform(coded=True)
+    sform, sform_code = orig.header.get_sform(coded=True)
+    header = resampled.header.copy()
+    header.set_qform(qform, int(qform_code))
+    header.set_sform(sform, int(sform_code))
+    for quatern in ['b', 'c', 'd']:
+        header['quatern_' + quatern] = orig.header['quatern_' + quatern]
```

Comment:
I'm pretty sure this is done in [`set_qform`](https://github.com/nipy/nibabel/blob/master/nibabel/nifti1.py#L1035).